packages/mdl-button/index.scss

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
+  display: inline-flex;
+  min-width: 64px;
+  height: 36px;
+  padding: 0;
+  border: none;
+  border-radius: 2px;
+  background: white;
+  box-sizing: border-box;
+  font-size: .875rem; /* 14sp */
+  font-weight: 500;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    height: 36px;
+    margin: 0;
+    padding: 8px 16px;
+    box-sizing: border-box;
+    pointer-events: none;
+  }
+
+  /* @TODO(Garbee) Make this shadow on hover for desktop */
+  &--raised {
+    min-width: 88px;
+    box-shadow:
+      0 2px 2px 0 rgba(0, 0, 0, .14),
+      0 3px 1px -2px rgba(0, 0, 0, .2),
+      0 1px 5px 0 rgba(0, 0, 0, .12);
+  }
+
+  &--primary {
+    color: var(--mdl-primary-color);
+
+    &.mdl-button--raised {
+      background-color: var(--mdl-primary-color);
+      color: white;
+    }
+  }
+
+  &--accent {
+    color: var(--mdl-accent-color);
+
+    &.mdl-button--raised {
+      background-color: var(--mdl-accent-color);
+      color: white;
+    }
+  }
+
+  &--dense {
+    height: 32px;
+    font-size: 13px;
+
+    .mdl-button__content {
+      height: 32px;
+    }
+  }
+
+  &:active:not(.mdl-button--accent):not(.mdl-button--primary) {
+    outline: none;
+    background-color: rgba(153, 153, 153, .4);
+  }
+
+  &:focus:not(:active) {
+    outline: none;
+
+    .mdl-button__content {
+      background-color: rgba(0, 0, 0, .12);
+    }
+  }
+
+  &--primary {
+    &:active:not(.mdl-button--raised) {
+      outline: none;
+      background-color: color(var(--mdl-primary-color) a(40%));
```

Comment:
any time a `var()` is used, we need to provide a duplicated property with a sass fallback, so that we follow @sgomes' theme guidance described in #4456